src/PrestaShopBundle/Resources/config/routing/admin/configure/shop_parameters/customer_groups.yml

```
@@ -0,0 +1,37 @@
+admin_customer_groups_index:
+  path: /
+  methods: [ GET ]
+  defaults:
+    _controller: 'PrestaShopBundle\Controller\Admin\Configure\ShopParameters\CustomerGroupsController::indexAction'
+    _legacy_controller: AdminGroups
+    _legacy_link: AdminGroups
```

```suggestion
    # _legacy_link: AdminGroups
```
You must keep this `_legacy_link` commented or the new page will automatically replace the old one, we don't ant that until the whole page (list + form) is fully implemented.

---

I didn't want to bother you with this on the first PR as it can be done later, but if you want you can use the `_legacy_feature_flag` feature if you don't want to comment the `_legacy_link`
It allows binding the legacy link feature (which automatically redirects to the new page) to a feature flag (configurable in the BO) thus you can switch between one or another

But it could be done later in a dedicated PR (if it's really small it could serve as an example for the community)